src/Components/LpButtonSection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './LpButtonSection.css'
+import { Link } from "react-router-dom"
+
+function LpButtonSection() {
+  return (
+    <div className='button-section-container'>
+
+      <Link style={{textDecoration: 'none'}}to="#"><button>Learn More</button></Link> 
```

Comment:
Where's this link go to?
```suggestion
      <Link style={{textDecoration: 'none'}} to="#"><button>Learn More</button></Link> 
```

Reply:
There is no where for it to go at the moment but when I build my next page I will link it to that page. So I will leave it this way for now.